README.md

```diff
@@ -69,52 +79,67 @@ DATABASES = {
 
 ### Django Rest Framework
 
- - Install our package requirements
+- Install our package requirements
 
 `pip install djangorestframework`
 
 `pip install pygments`
 
- - Add INSTALLED_APPS
+- Add INSTALLED_APPS
 
-```json
-INSTALLED_APPS = [
-    'rest_framework',
-]
+```js
+INSTALLED_APPS = ["rest_framework"];
```

Comment:
note: I resolve json to js

because this syntax is javascript.

and I format it from my vscode. appear this~

Reply:
Ok